### PR TITLE
feat(generators): Update the generator to setup projects using babel v7+

### PR DESCRIPTION
**Pull request by risheepatel**

### CODE REVIEW

File: __tests__/app.js [hunk 1]
```diff
@@ -34,18 +34,20 @@ describe('generator-seedrs-npm:app', () => {
...
+          '@babel/core': '7.1.2',
+          '@babel/cli': '7.1.2',
+          '@babel/plugin-proposal-object-rest-spread': '7.0.0',
+          '@babel/preset-env': '7.1.0',
           '@seedrs/eslint-config-seedrs-base': '^1.1.1',
-          'babel-cli': '^6.26.0',
-          'babel-core': '^6.26.3',
-          'babel-plugin-transform-object-rest-spread': '^6.26.0',
-          'babel-preset-env': '^1.7.0',
+          'babel-core': '^^7.0.0-bridge',
```
Comment (kenleytomlin) [inline __tests__/app.js]: @risheepatel I think this should only have a single pin not two

File: generators/app/templates/package.json.tpl [hunk 2]
```diff
@@ -4,18 +4,20 @@
...
+    "@babel/core": "7.1.2",
+    "@babel/cli": "7.1.2",
+    "@babel/plugin-proposal-object-rest-spread": "7.0.0",
+    "@babel/preset-env": "7.1.0",
     "@seedrs/eslint-config-seedrs-base": "^1.1.1",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.3",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
+    "babel-core": "^^7.0.0-bridge",
```
Comment (kenleytomlin) [inline generators/app/templates/package.json.tpl]: Also I think the pin is duplicated unless I am mistaken

**Review by kenleytomlin — CHANGES_REQUESTED** [review]
I think the package.json contains two issues with the double pins (^)

File: __tests__/app.js (hunk 1 above)
Comment (risheepatel) [reply __tests__/app.js]: Ah good point. Sloppy copy and paste there. 👍

**Comment by risheepatel** [comment]
@kenleytomlin That's updated now

**Review by kenleytomlin — APPROVED** [review]
👍 🚀

**Pull request merged**